### PR TITLE
Add Facebook for the Front End Engineering position

### PR DESCRIPTION
**Pull request by aadsm**
## Add Facebook for Front End Engineer

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [x] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

When interviewing with Facebook for a Front End Engineering position all questions are practical ones that people in this role had to solve in the past. For this reason I believe it  passes the criteria defined.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,6 +262,7 @@ Also check out [No Whiteboards](https://nowhiteboards.io) to search for jobs at
 - [Euro Payment Group](http://www.europaymentgroup.com) | Frankfurt, Germany | Take-home project followed by face to face interview
 - [Exoscale](https://exoscale.ch) | Bern, Switzerland | Take-home project. Discussion and presentation. Then entire team meet.
 - [F(x)](https://fdex.com.br) | São Paulo, Brazil | Skype interview, Take-home project and onsite interview to evaluate the candidate
+- [Facebook (Front End Engineer)](https://www.facebook.com/careers/jobs) | Tel Aviv, Israel; London, UK; US: Menlo Park, CA; Seattle, WA; Wasthington, DC | Initial phone code interview, then on site interview (or remote when not possible) with code and design / architecture interviews. All questions are based on actual problems the company has solved in the past.
```
Comment (poteto) [inline README.md:265]: There's a typo here: `s/Wasthington/Washington`

Comment (aadsm) [reply README.md:265]: Thank you, I can't believe I made a typo when I only wrote a few of words 🤦.

**Comment by poteto** [comment]
Thanks for contributing! bors r+

**Comment by poteto** [comment]
Thanks for contributing! 

bors r+

**Pull request merged**